CMSgov/mdct-mcr
# [MDCT-1965, 1975-1978] - Quality measures

## Description
Fixes everything related to Quality Measures

### How to test
./dev local
Create a program
Go to section D
Create a quality Measure
See that it creates a quality measure for you to fill out
Double check that the modal hydrates with your answers
Open the quality measure drawer
See that theres associated data at the top and no plans at the bottom 
Close the drawer
Back track and add as many plans as you want in Section A
Go back to Quality Measures
Open the drawer
Add in your responses to the new plans at the bottom
See the status update in the entity card

Now delete a plan, see that its still ok
Add a new plan, see that its not ok
Create new measures and make sure all the possible other fields can be entered, hydrated, and reflected
Delete the new measures if you want
navigate away and come back and see things are still deleted

### Changed Dependencies


## Code author checklist

## Reviewer checklist (two different people)
